docs/sample-solutions/fasttrack/solution-template.md

```diff
@@ -18,7 +18,7 @@ You should consider this solution for the following use cases:
 
 The solution diagram below is an example of this solution:
 
-![Description][architecture]
+![Hybrid HPC Architecture](media\AzureBatchArchitecture.png)
```

Comment:
This image doesn't match our style. It should show the infrastructure that is deployed and use the standard Azure service icons.